### DISCUSSION TITLE
Restore Windows Vista/7/8 borders + Remove the fade on Inactive Captions for Windows 11

### DISCUSSION THREAD

**Discussion by TheDolphinSlayer** [Maplespe/DWMBlurGlass#246] [Ideas]
Just an idea, there are UDWM patches out there + Windhawk mods for the borders, but they don't work too well with DWMBlurGlass (for obvious reasons). There is mods for the inactive captions for windows 10 but they don't work for windows 11, so I figured if we can't get text glow maybe we can get that instead? 

Here is an example with the windhawk restore borders mod (the blur doesn't apply to the borders).
Also included an example at the bottom  of what I meant by removing the faded inactive captions (that were added in windows 10/stayed with win11)
![image](https://github.com/Maplespe/DWMBlurGlass/assets/73983610/fab8ac2c-cc4e-4947-8953-baef421d5d99)
![image](https://github.com/Maplespe/DWMBlurGlass/assets/73983610/d37eace2-584e-4dd9-ada6-93e880aea004)

**Comment by Sand216** [comment]
Bump, specifically for the borders. Is there any way window borders could be implemented on the DWM side, or is it out of scope for this project? Currently, the only way I could find to restore window borders in Windows 11 is to use a high contrast theme, which means the code for borders is somewhere...

**Comment by 7kt4** [comment]
This is a rly weird problem that I have spent so much time on. With the advanced thick frame mod (that injects into DWM), Windows 10's unmodded DWM knows to render the (not transparent) BlurBehind into the newly thick-framed window, as you can see in the screenshot without transparency.  I thought that DWMBlurGlass works by simply making the normal blurbehind blurry again. But...even though the entire new frame has the blurbehind enabled, it does not get blurred with DWMBG. I don't know how that is possible?

Oh, and one more thing. The right and bottom frames DO get blurred with DWMBG. Just not the left and top frame. Really weird. I guess DWMBG's method of taking the window's top left position is not compatible with this mod, but its method of finding the bottom and right positions are?

![image](https://github.com/Maplespe/DWMBlurGlass/assets/84914212/07063fca-a948-4158-ad67-98fb5c629056)
![image](https://github.com/Maplespe/DWMBlurGlass/assets/84914212/a8bc80e4-82b2-4740-abe3-5b2df7d45a4b)

**Comment by Sand216** [comment]
@TheDolphinSlayer Also, what theme/msstyles are you using in the picture for the aero titlebar buttons and elements on Windows 11? I’ve tried Aero10 in a VM but it has its incompatibilities on Windows 11.

**Reply by TheDolphinSlayer** [reply]
Aero10 modified to work with windows 11, I can send but its old by 2 months
https://www.mediafire.com/file/kdsn3737de5184w/Aero_11.zip/file
